src/Apps/ArtAdvisor/06-Discovery-Neartext/components/Concepts.tsx

```diff
@@ -0,0 +1,39 @@
+import { FC } from "react"
+import _ from "lodash"
+import { Box, Button, Spacer, Text, TextArea } from "@artsy/palette"
+
+interface ConceptsProps {
+  conceptText: string
+  setConceptText: (text: string) => void
+  setConcepts: (concepts: string[]) => void
+}
+export const Concepts: FC<ConceptsProps> = props => {
+  const { conceptText, setConceptText, setConcepts } = props
+  return (
+    <Box py={2}>
+      <Text variant={"lg-display"}>Concepts</Text>
+      <Spacer y={1} />
+
+      <Text>Enter one or more "user concepts", one concept per line:</Text>
+      <Spacer y={1} />
+
+      <TextArea
+        placeholder="street art, e.g."
+        defaultValue={conceptText}
+        rows={5}
+        onChange={({ value }) => setConceptText(value)}
+      ></TextArea>
+      <Spacer y={1} />
+
+      <Button
+        onClick={e => {
+          let concepts = (conceptText || "").split("\n")
+          concepts = _.reject(concepts, c => _.isEmpty(c.trim()))
```

Comment:
Taking the newline-delimited concept text from the `textarea` and turning it an array of non-empty concepts here. (Weaviate is very fussy about being sent an empty list, or empty concepts)